### PR TITLE
Paranoid: Addressing "the joy of discovery"

### PR DESCRIPTION
**Pull request by lintool**
Re: https://twitter.com/drjessehoey/status/1195343091139567616

### CODE REVIEW

File: paranoid.md [hunk 1]
```diff
@@ -59,6 +59,16 @@ Wow, that's amazing.
...
 
+The final cluster of objections is what I would characterize as "intrinsic motivation":
+instances include, for example, "for the simple joy of discovery", or "I'm not paranoid because I don't care about precedence, publications, h-indexes, citations, etc.", or "I'm not doing this for external validation".
+In some cases, these sentiments are expressed by people who are in the enviable position of not needing to care, for example, post-tenure academics.
+This strikes me as disingenuous if offered as serious advice, i.e., "_you_ are in a position where you don't need to run the publication rat race, but advising students that publication count doesn't matter _at all_"?
+When these sentiments are expressed, for example, by pre-tenure academics, I really hope they are cognizant of the consequences of their approach (on promotion and tenure decisions, etc.).
+I've heard many times: "if this tenure thing doesn't work out, I'll just do _X_, but until then I'll do what I to do".
+And they truly mean it.
+I genuinely admire such individuals.
+I just haven't attained such a level of enlightenment.
```
Comment (emanlapponi) [inline paranoid.md]: I think this statement sounds condescending and impertinent, completely negating the genuine admiration expressed in the previous sentence.

File: paranoid.md [hunk 2]
```diff
@@ -59,6 +59,16 @@ Wow, that's amazing.
 Perhaps you're just so insightful that you see something all of us have missed.
 It may be true, or it's just _hubris_.
 
+The final cluster of objections is what I would characterize as "intrinsic motivation":
+instances include, for example, "for the simple joy of discovery", or "I'm not paranoid because I don't care about precedence, publications, h-indexes, citations, etc.", or "I'm not doing this for external validation".
+In some cases, these sentiments are expressed by people who are in the enviable position of not needing to care, for example, post-tenure academics.
+This strikes me as disingenuous if offered as serious advice, i.e., "_you_ are in a position where you don't need to run the publication rat race, but advising students that publication count doesn't matter _at all_"?
+When these sentiments are expressed, for example, by pre-tenure academics, I really hope they are cognizant of the consequences of their approach (on promotion and tenure decisions, etc.).
+I've heard many times: "if this tenure thing doesn't work out, I'll just do _X_, but until then I'll do what I to do".
```
Comment (sdht0) [inline paranoid.md]: missing word: "until then I'll do what I __ to do"?

File: paranoid.md (hunk 1 above)
Comment (lintool) [reply paranoid.md]: This is interesting. A genuine attempt at deprecation can be interpreted as snide. I'll think about how to rephrase.

File: paranoid.md (hunk 2 above)
Comment (lintool) [reply paranoid.md]: Thanks for the catch.

**Comment by lintool** [comment]
Hearing no further comments, merging.

**Pull request merged**